code.gs

```diff
@@ -1513,10 +1514,14 @@ function beautifyLabel (label) {
     case 'WORK_FAX':
     case 'GOOGLE_VOICE':
     case 'PAGER':
+    // Email labels
     case 'HOME_EMAIL':
     case 'WORK_EMAIL':
+    // Event labels
+    case 'OTHER': // Offial Google label for events.
+    // Fake labels
     case 'OTHER_EMAIL': // Fake label for output.
-    case 'OTHER_PHONE': // Fake label for output,
+    case 'OTHER_PHONE': // Fake label for output.
```

Comment:
I'd just move `OTHER_PHONE` and `OTHER_EMAIL` in the email and phone groups, since they already have inline comments explaining why they are different from the other ones.

Also I am not exactly sure what `// Offial Google label for events.` is supposed to mean: if you wanted to specify that it's not a fake label I'd remove it since the only two fake ones are well commented and all the other ones are "Official" by exclusion.

Reply:
I forgot to remove this comment. I had put this before I grouped those labels.